.github/ISSUE_TEMPLATE/documentation_or_example.md

```diff
@@ -7,3 +7,6 @@ assignees: ""
 ---
 
 Give us a description of what you're looking for!
+
+**How does this increase our chances of an iCrawl Acquisition?**
+Please explain how this will increase our relative value to iCrawl in order to further our goal of acquisition
```

Comment:
```suggestion
**How does this increase our chances of an iCrawl Acquisition?**
Please explain how this will increase our relative value to iCrawl in order to further complete our final goal and life purpose of the acquisition
```